<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cherygova-cg16 TW</string>
  </property>
  <layout class="QGridLayout" name="WidgetLayout">
   <item row="1" column="1">
    <spacer name="VSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>215</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="HSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>361</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="Control">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="TabGeneralLayout">
       <item row="2" column="2">
        <widget class="QRadioButton" name="CircleButton">
         <property name="text">
          <string>Circle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ObjectLabel">
         <property name="text">
          <string>Object:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="AlgoLabel">
         <property name="text">
          <string>Algorithm:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="AlgoComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bresenham</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wu</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4xOGSSAA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16xOGSSAA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64xOGSSAA</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="SegmentButton">
         <property name="text">
          <string>Seg&amp;ment</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QSpinBox" name="StepSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="StepLabel">
         <property name="text">
          <string>Grid step:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="BaseLinesCheckBox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Show base lines:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabObjects">
      <attribute name="title">
       <string>Objects</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SegmentButton</tabstop>
  <tabstop>CircleButton</tabstop>
  <tabstop>AlgoComboBox</tabstop>
  <tabstop>StepSpinBox</tabstop>
  <tabstop>BaseLinesCheckBox</tabstop>
  <tabstop>Control</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
